data-es6.js

```diff
@@ -9649,7 +9649,9 @@ exports.tests = [
         var get = [];
         var p = new Proxy({}, { get: function(o, k) { get.push(k); return o[k]; }});
         Object.getOwnPropertyDescriptor(RegExp.prototype, 'flags').get.call(p);
-        return get + '' === "global,ignoreCase,multiline,unicode,sticky";
+        return get.indexOf('global') !== -1 &&
+               get.indexOf('ignoreCase') !== -1 &&
+               get.indexOf('multiline') !== -1;
```

Comment:
this now no longer tests for sticky or unicode, and also no longer tests the order - both of which are important.

Reply:
Unicode and sticky are both ES6 extensions and not part of the feature being tested.

I wasn't aware that order was significant; I'll update the PR.

Reply:
This is a test for flags; it requires testing all flags.

Reply:
This is a test for "Proxy, internal 'get' calls" and how they act on `RegExp.prototype.flags`, not for completeness of the implementation's RegExp support.

If it should not fail when `dotAll` is present (which is the bug in issue #1208), then why should it fail when `unicode` and `sticky` are absent?

Reply:
I think this test can be written such that it does not fail when dotAll is present or not.